scss/pages/_civicrm-dashboard.scss

```diff
@@ -1,7 +1,46 @@
+/* stylelint-disable selector-max-id  */
+/* stylelint-disable selector-max-compound-selectors */
+
 .___outside-namespace {
   .page-civicrm-dashboard,
   // targets root `page-civicrm` page and doesn't target child `page-civicrm-event` pages (for example):
   .page-civicrm:not([class*=' page-civicrm-']) {
     @include case-page;
   }
+
+  .civicase__crm-dashboard {
+    &.ui-tabs,
+    .tab-content {
+      background: none;
+    }
+
+    &__tabs.affix {
+      position: fixed;
```

Comment:
This is already present https://github.com/compucorp/uk.co.compucorp.civicase/blob/ea9c9261e10671d3ada9df51820580a72300617c/scss/components/_crm-dashboard.scss#L8

So do we need `position: fixed` and `z-index` again?

Reply:
We need to. The styles for affixing an element only work inside `#bootstrap-theme` and this is outside.